front/admin/cli.ts

```diff
@@ -347,12 +347,13 @@ const dataSource = async (command: string, args: parseArgs.ParsedArgs) => {
           `DataSource not found: wId='${args.wId}' provider='${args.provider}'`
         );
       }
+      if (!dataSource.connectorId) {
+        throw new Error(
+          `DataSource does not have a connector: wId='${args.wId}' connectorId='${args.connectorId}'`
```

Comment:
`DataSource does not have a connector: wId='${args.wId}' dataSource.id='${dataSource.id}'`